source/extensions/filters/network/zookeeper_proxy/zookeeper_filter.h

```diff
@@ -63,7 +67,7 @@ struct ZooKeeperProxyStats {
  */
 class ZooKeeperFilterConfig {
 public:
-  ZooKeeperFilterConfig(const std::string &stat_prefix, uint32_t max_packet_bytes, Stats::Scope& scope);
+  ZooKeeperFilterConfig(const std::string& stat_prefix, uint32_t max_packet_bytes, Stats::Scope& scope);
```

Comment:
huh how come format test misses this? can we fix that test so it catches this?

Reply:
Yeah I'm also puzzled... it's also messing up import headers. Example in `zookeeper_config.cc` from (main header at top)
```
#include "extensions/filters/network/zookeeper_proxy/zookeeper_config.h"

#include "envoy/config/filter/network/zookeeper_proxy/v1alpha1/zookeeper_proxy.pb.validate.h"
#include "envoy/registry/registry.h"
#include "envoy/server/filter_config.h"

#include "common/common/logger.h"

#include "extensions/filters/network/zookeeper_proxy/zookeeper_filter.h"
```

to (headers always alphabetical)
```
#include "envoy/config/filter/network/zookeeper_proxy/v1alpha1/zookeeper_proxy.pb.validate.h"
#include "envoy/registry/registry.h"
#include "envoy/server/filter_config.h"

#include "common/common/logger.h"

#include "extensions/filters/network/zookeeper_proxy/zookeeper_config.h"
#include "extensions/filters/network/zookeeper_proxy/zookeeper_filter.h"
```

Reply:
I think the trailing \ on https://github.com/envoyproxy/envoy/blob/master/source/extensions/filters/network/zookeeper_proxy/zookeeper_filter.h#L51 is disabling clang-format for this code. Could you try removing it and running clang-format to see if that fixes it?

Reply:
🤦‍♂️ yep that was it! Also fixes a few of other formatting things. Thanks @snowp!